### PR TITLE
refactor(common): 13882 migration from deprecated styles to fresh

### PR DESCRIPTION
**Pull request by Akiyamka**
We have update of UI kit design - now we have much more header styles and it separate component.
This commit migrate codebase to new <Header /> component

### CODE REVIEW

File: src/features/user_profile/components/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ export function LoginForm() {
           <LoadingSpinner message={i18n.t('login.logging_in')} />
         </div>
       )}
-      <Text type="heading-xl">{i18n.t('login.log_in')}</Text>
+      <Heading type="heading-06">{i18n.t('login.log_in')}</Heading>
```
Comment (propakov) [inline src/features/user_profile/components/LoginForm/LoginForm.tsx]: hm, it differs a bit from what we had. 
is it ok? 
before it was closer to the middle of the screen 
image

Comment (propakov) [reply src/features/user_profile/components/LoginForm/LoginForm.tsx:101]: before login inputs were closer to the middle of the screen 
Screenshot 2023-01-16 at 18 02 53

File: src/features/user_profile/components/SettingsForm/SettingsForm.tsx [hunk 2]
```diff
@@ -113,8 +113,9 @@ export function SettingsForm() {
         <div className={s.notFlexParent}>
           <div className={s.flexWrap}>
             <div className={s.profileWrap}>
-              <Text type="heading-xl">{i18n.t('profile.profileSettingsHeader')}</Text>
-
+              <Heading type="heading-06">
```
Comment (propakov) [inline src/features/user_profile/components/SettingsForm/SettingsForm.tsx]: same moment here, padding differs drastically
not sure, but is it ok?

Comment (Akiyamka) [reply src/features/user_profile/components/SettingsForm/SettingsForm.tsx]: hmm, thanks, i will recheck this

**Review by propakov — APPROVED** [review]
apart from heading-06 paddings code LGTM. 
so only this moment need to be checked. 🚀

File: src/features/user_profile/components/SettingsForm/SettingsForm.tsx (hunk 2 above)
Comment (Akiyamka) [reply src/features/user_profile/components/SettingsForm/SettingsForm.tsx]: fixed

File: src/features/user_profile/components/LoginForm/LoginForm.tsx (hunk 1 above)
Comment (Akiyamka) [reply src/features/user_profile/components/LoginForm/LoginForm.tsx]: fixed

**Pull request merged**